lib/marc_to_argot/macros/duke/items.rb

```diff
@@ -226,11 +226,14 @@ def extract_holdings
               holdings << holding
               # acc << holding.to_json if holding.any?
             end
+            ctx.clipboard[:holdings_present] = !holdings.empty?
```

Comment:
Not sure this is the right test for "Available" per the document level usage above -- that will show "Available" if `:holdings_present` is set to `true` in the clipboard, but this line means that will be true if _holdings data is present_, which I believe is different from "at least one holdings has a status of available"